Readme: Mention message definition update script (#2)
PX4/px4_msgs: README.md

## How are these messsage definitions generated?

> When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository changes, a Jenkins-based CI/CD automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by the [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py) script. One can also use this script to generate its own ROS message definitions for new or modified uORB messages.

Reviewer: ```suggestion
When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository change, a Jenkins-based CI/CD automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by the [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py) script. One can also use this script to generate its own ROS message definitions for new or modified uORB messages.
```

Reviewer: It's not in Jenkins anymore, but rather in Github Actions.

Author: Aha! I found it still in the Jenkinsfile, but maybe that's just kept around but not running. And thanks for spotting the typo.

## How are these messsage definitions generated?

> When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository changes, a Jenkins-based CI/CD automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by a script [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py). You can also use this script to generate your own ROS message definitions, for instance if you have a custom PX4 firmware fork with modified uORB definitions.

Reviewer: ```suggestion
When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository changes, a Jenkins-based CI/CD automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by the [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py) script. One can also use this script to generate its own ROS message definitions for new or modified uORB messages.
```

Author: I agree. Are you okay with keeping the "you can" instead of "one can"? I feel it's more clear/direct.

Reviewer: On this kind of things I do prefer the 3rd person.

Author: :+1:

> A future improvement will also include tagging and releasing this repository according to the tags/releases of the PX4 Firmware, for a more tidy coupling between the message definitions.

Reviewer: We should remove this from the README and put it under an issue instead. Can you please remove it? Thanks!

Author: Absolutely!
